<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>map</class>
 <widget class="QWidget" name="map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Map</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>1180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1180</width>
     <height>50</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pb_route">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Display Route</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_animate">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Animate Route</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_backedges">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Display Back/Cross Edges</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_all">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Display All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_exit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::ApplicationExit"/>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1180</width>
     <height>570</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1180</width>
     <height>570</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1180</width>
     <height>570</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap resource="logos.qrc">:/logos/usa-map.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logos.qrc"/>
 </resources>
 <connections/>
</ui>
